src/main/resources/attackvectors/BlindSQLInjectionVulnerabilityPayload.properties

```diff
@@ -0,0 +1,29 @@
+BLIND_SQL_INJECTION_PAYLOAD_LEVEL_1=As blind SQL injection doesn't show any internal details like errors or table data hence it allows a hacker to check if the web application is vulnerable to SQL injection by sending true and false statements.<br/>\
+  How to exploit this level?<br/>\
+  Insert those in textbox<br/>\
+  For true statement<br/>\
+  <i>100 OR 2=2</i><br/>\
+  Expected result<br/>\
+  <i>{ "isCarPresent": true }</i><br/>\
```

Comment:
Can you bold out the question like 

<b>How to exploit this level?</b><br/>\
Enter following statement in textbox: <br/>\
<i>100 OR 2=2</i><br/>\
Expected result<br/>\
<i>{ "isCarPresent": true }</i><br/>\

Reply:
can you please work on better styling of the string ?